requirements.txt

```diff
@@ -0,0 +1,2 @@
+jsonschema
```

Comment:
I swore I saw this as a dependency in the original code, but of course it's not there. Removed.